tests/test_clients.py

```diff
@@ -101,7 +101,7 @@
 async def test_title_search(paper_attributes: dict[str, str]) -> None:
     async with aiohttp.ClientSession() as session:
         client_list = list(ALL_CLIENTS)
-        client_list.remove(RetrationDataPostProcessor)
+        client_list.remove(RetractionDataPostProcessor)
```

Comment:
Can you add an inline comment why we remove this processor here in the tests?

Alternately, what do you of something like this at the module level?

```python
# Exclude RetrationDataPostProcessor because abc123
TEST_CLIENTS = {ct for ct in ALL_CLIENTS if ct not in {RetrationDataPostProcessor}}
```